docs/Testing Crossmodel.md

```diff
@@ -0,0 +1,47 @@
+# Tests: CrossModel Community Edition
+
+## Getting started
+
+The testing framework for this project is Jest. The dev NPM packages are _jest_ and _ts-jest_.
+
+## Setting up the tests on your system
+
+Run this command in your terminal: `yarn`
+
+It is really recommended to run the Jest tests using the VS Code extension: _Jest_ by publisher Orta. With these extensions, tests will automatically run when updated and it is very easy to run the tests from the VS Code GUI (using the button immediately left of the test in question).
+
+## Writing tests
+
+Unit tests can be placed in the same directory as the TypeScript files they test. It is also possible to add them to the `crossmodel/tests/` directory. Functional and integration tests using _Cucumber_ will be located in the `crossmodel/tests/` directory as well.
```

Comment:
I wouldn't mention Cucumber here, since we are not using it, and not sure whether we will.